lib/buildingsync/helpers/Model.hvac.rb

```diff
@@ -0,0 +1,210 @@
+class OpenStudio::Model::Model
```

Comment:
@TMaile this seems like it should be in OpenStudio Standards, did you write this from scratch or get it somewhere else?  We should not be adding code to the `OpenStudio::Model` module in this gem, all code specific to this gem should go into the `BuildingSync` module

Reply:
@macumber we got this helper file from BRICR measure `"create_typical_building_from_model_bricr`". I did not find this method `"add_cbecs_hvac_system" `with this name in any of the other gems. Did I miss it somewhere or should we just place it into `BuildingSync`.

Reply:
I think we should make another da specific branch of the [openstudio-model-articulation-gem](https://github.com/NREL/openstudio-model-articulation-gem/tree/develop/lib/measures/create_typical_building_from_model) to put this in.  The BuildingSync gem should not be tied to any one standard (BRICR is specific to one California based standard but BuildingSync is not).

Reply:
@mrigeshgupta will do this as described in #12